examples/offline_inference_neuron.py

```diff
@@ -0,0 +1,33 @@
+import os
+from vllm import LLM, SamplingParams
+
+# Sample prompts.
+prompts = [
+    "Hello, my name is",
+    "The president of the United States is",
+    "The capital of France is",
+    "The future of AI is",
+]
+# Create a sampling params object.
+sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
+
+os.environ['MASTER_ADDR'] = 'localhost'
+os.environ['MASTER_PORT'] = '12355'
+os.environ['NEURONX_DUMP_TO'] = os.path.join(os.getcwd(), "_compiler_cache")
+os.environ["NEURON_CC_FLAGS"] = " -O1 "
```

Comment:
Are these env variables necessary to run Inf2/Trn1? If so, can we set them internally in vLLM?

Reply:
Since all these are optional, i removed these optional settings for simplicity.